src/test/resources/org/csanchez/jenkins/plugins/kubernetes/pipeline/jenkinsSecretHidden.groovy

```diff
@@ -0,0 +1,12 @@
+podTemplate(containers: [
+        containerTemplate(name: 'busybox', image: 'busybox', ttyEnabled: true, command: '/bin/cat'),
+]) {
+    node (POD_LABEL) {
+      semaphore 'pod'
+      stage('Run') {
+        container('busybox') {
+          sh "true"
```

Comment:
```suggestion
          sh 'env | sort'
```

(doing this also outside `container`) and I suspect you will find that this fix does not suffice. Try reverting the change to `KubernetesSlave` and adding

```diff
diff --git a/src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/SecretsMasker.java b/src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/SecretsMasker.java
index 87d6b95a..127c002b 100644
--- a/src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/SecretsMasker.java
+++ b/src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/SecretsMasker.java
@@ -139,6 +139,7 @@ public final class SecretsMasker extends TaskListenerDecorator {
             }
             Pod pod = slave.getTemplate().build(slave);
             Set<String> values = new HashSet<>();
+            values.add(c.getJnlpMac());
             LOGGER.finer(() -> "inspecting " + pod);
             for (Container container : pod.getSpec().getContainers()) {
                 Set<String> secretContainerKeys = new TreeSet<>();
@@ -191,7 +192,7 @@ public final class SecretsMasker extends TaskListenerDecorator {
                     // TODO issue a warning if we did not find values for all of secretContainerKeys
                 }
             }
-            return values.isEmpty() ? null : values;
+            return values;
         }
 
     }
```

Reply:
Looks good, though I had to keep the change in `KubernetesSlave` as the `TaskListener` there isn't decorated with `SecretMasker` at this point.